k8s/contiv-vpp/values.yaml

```diff
@@ -14,7 +14,7 @@ contiv:
   serviceLocalEndpointWeight: 1
   disableNATVirtualReassembly: True
   ipamConfig:
-    contivCIDR: "10.1.0.0/16"
+    contivCIDR: "10.1.0.0/15"
```

Comment:
`.1` the second octet will be ingnored due to mask used, rather use `10.2.0.0/15`

Reply:
I think this needs to change to 10.0.0.0/14 now, right?

Reply:
yes that's another option